src/coreclr/utilcode/ex.cpp

```diff
@@ -79,7 +96,12 @@ void Exception::Delete(Exception* pvMemory)
         return;
     }
 
+#ifdef DACCESS_COMPILE
+    pvMemory->~Exception();
+    DeleteDbiMemory(pvMemory);
+#else
     ::delete((Exception *) pvMemory);
+#endif
```

Comment:
Can this be:

```suggestion
    Exception *ex = (Exception *)pvMemory;
#ifdef DACCESS_COMPILE
    delete ex;
#else
    ::delete(ex);
#endif
```

Reply:
The case to ex is redundant since pvMemory is all ready Exception *.